apps/portals/adknowledgeportal/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
The HTML for each portal is identical. All of the shared content is injected via template (see `headContent`, `gtmNoscript`)